### ISSUE TITLE
Question:Custom Model detection

### ISSUE THREAD

**Issue by junmcenroe** [Ma-Dan/YOLOv3-CoreML#23]
Dear Author

I am trying to convert the pretrain-custom model(960x960, 22classes)
Convert was succeeded and detect with Keras model correctly.
Then convert to mlmodel is also seems to be success with following.
In Xcode,
Input :Image (Color 960 × 960)
Output3:MultiArray (Float32 81 × 120 × 120)
Output2:MultiArray (Float32 81 × 60 × 60)
Output1:MultiArray (Float32 81 × 30 × 30)
And I changed the following static data in Swift, and label data
**
public static let inputWidth = 960
public static let inputHeight = 960

let numClasses = 22

let gridHeight = [30, 60, 120]
let gridWidth = [30, 60, 120]
**
After running the program , no detection happened.Actually, 
Tracing the data of tc
let tc = Float(featurePointer[offset(channel + 4, cx, cy)])
always tc value is negative value, so after sigmoid conversion , mostly zero.
then no detection as finally.
Are there any missing points to run the custom model?
